src/AppInstallerSharedLib/Public/winget/JsonUtil.h

```diff
@@ -33,6 +33,9 @@ namespace AppInstaller::JSON
 
     template<>
     std::optional<std::vector<std::string>> GetValue<std::vector<std::string>>(const Json::Value& node);
+
+    template<>
+    std::optional<std::map<std::string, bool>> GetValue(const Json::Value& node);

```

Comment:
Why does this one not have the type after `GetValue`? Like how above it is `GetValue<std::vector<std::string>>` instead of just `GetValue`?